climada/hazard/tc_surge_geoclaw.py

```diff
@@ -0,0 +1,1451 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU Lesser General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Inundation from TC storm surges, modeled using the library GeoClaw
+
+This module requires a Fortran compiler (such as gfortran) to run!
+"""
+
+import __main__
+import contextlib
+import datetime as dt
+import importlib
+import logging
+import re
+import pathlib
+import site
+import subprocess
+import sys
+import warnings
```

Comment:
Is this module needed? I have seen it in a previous pull request (wildfires), but it ended up not being used at all.

Reply:
It's needed to suppress irrelevant warnings when reading the output of GeoClaw:
https://github.com/CLIMADA-project/climada_python/blob/964d976f096bf991bb14d94df156ba0d811518e9/climada/hazard/tc_surge_geoclaw.py#L561-L564